src/style.css

```
@@ -1,3 +1,124 @@
+html {
+  --url-img-star: url(./images/star.svg);
+  --url-img-star-active: url(./images/star-active.svg);
+  --text-color-main: #060b35;
+  --text-color-secondary: #616070;
+  --blue-accent: #00acdc;
+  --white: #fff;
+}
+
 body {
   margin: 0;
+  font-family: "Roboto", Arial, sans-serif;
+  font-style: normal;
+}
+
+.card {
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+}
+
+.card__image {
+  display: block;
+  margin: 0 auto;
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  margin-top: 40px;
```

```suggestion
.card__title {
  margin: 0;
  margin-top: 40px;
```

Almost done, please reset the default margin for `<h1>` tag. You can reset it here but better reset it above or in a separate reset.css file:

```
h1 {
  margin: 0;
}
```